src/main/kotlin/dev/arbjerg/ukulele/audio/TrackQueue.kt

```diff
@@ -17,5 +17,14 @@ class TrackQueue {
         queue.removeAll(list)
         return list
     }
-
-}
\ No newline at end of file
+    fun getQueue(): List<AudioTrack> {
+        return queue
+    }
+    fun getDuration(): Long {
```

Comment:
Want to save 6 lines of code?
`fun getDuration() = queue.sum { it.info.length }`